src/Styles.re

```diff
@@ -30,7 +29,7 @@ let app =
 let textOnlyModalText = style([padding(em(2.))]);
 let totalRaisedText = size => style([fontSize(em(size))]);
 
-let copyButton = style([float(`right), zIndex(50)]);
+let copyButton = style([Css.float(`right), zIndex(50)]);
```

Comment:
I was getting a warning that I was overshadowing the name `float` which is also a type not just for css. This fixes that warning.